src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -147,6 +151,84 @@ const char* GetCalendarName(CalendarId calendarId)
 
 /*
 Function:
+GetCalendarName
+
+Gets the associated CalendarId for the ICU calendar name.
+*/
+CalendarId GetCalendarId(const char* calendarName)
+{
+	if (strcmp(calendarName, "gregorian") == 0)
```

Comment:
is it possible the calendar name can be in different casing? I mean you can get "Gregorian"? if so, then the string comparison should be case insensitive